### PR TITLE
Correct grammar in #who_can_help

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR corrects a typo / grammatical error in https://www.coronawarn.app/de/faq/#who_can_help

"Um Ihnen die Suche nach dem zuständigen Gesundheitsamt zu erleichtern, hat das RKI **eine** entsprechendes Tool zur Verfügung gestellt."

corrected to "... ein entsprechendes Tool ...".

## Reference

https://www.duden.de/rechtschreibung/Tool (Substantiv, Neutrum)

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Many thanks!

**Comment by MikeMcC399** [comment]
@dsarkar 

The deployment has failed.
See https://github.com/corona-warn-app/cwa-website/actions/runs/654704529

I guess you could either re-trigger the action or just wait until the next PR merge for it to catch up.

It looks like it was a temporary Internet issue on GitHub. 
```
E: The repository 'https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04  Release' no longer has a Release file.
Error: Process completed with exit code 100.
```

The same line of the workflow executed correctly 4 hours ago!?

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I think @dsarkar should re-trigger it to make sure that this was a temporary issue, or?

**Pull request merged**
